config/settings.yml

```diff
@@ -1090,7 +1090,7 @@
       :poll_method: :normal
       :starting_timeout: 10.minutes
       :stopping_timeout: 10.minutes
-      :systemd_enabled: true
+      :systemd_enabled: false
```

Comment:
Since we apparently haven't been testing with systemd enabled on appliances for jansa (probably since https://github.com/ManageIQ/manageiq/pull/19556) I think disabling this for jansa is safest

Reply:
For context, we were just looking at an appliance where workers were spawned from evmserverd.service but when we inspected workers, they indicated they were systemd workers and could be the reason that we never were stopping the workers, because they were started from spawn (old way) and we were trying to stop them from systemd (which is not how they started) so nothing happened and it continually tried to stop them in the same way...

```
irb(main):002:0> MiqWorker.find(13504).systemd_worker?
=> true
```